src/index.js

```diff
@@ -48,10 +47,30 @@ const clickhandler = () => {
     const userAttempt = document.querySelector('.username-input').value
     const pwAttempt = document.querySelector('.pw-input').value
     const loginOutcome = hotel.checkValidation(userAttempt, pwAttempt)
-    domUpdates.determineDash(loginOutcome)
+    determineDash(loginOutcome)
   }
 }
 
+const determineDash = (outcome) => {
+  domUpdates.todaysDate = todaysDate
```

Comment:
This look great, but if you feel inclined a switch statement would also work really well here. Just if you wanna mix it up a bit.